src/System.IO.Pipelines/PipeFactory.cs

```diff
@@ -99,7 +108,11 @@ public IPipeWriter CreateWriter(IPipeWriter writer, Func<IPipeReader, IPipeWrite
         public IPipeReader CreateReader(IPipeReader reader, Func<IPipeReader, IPipeWriter, Task> produce)
```

Comment:
Sorry, because this isn't really related to the PR at all, but @davidfowl why would you ever need this API in a world where you already have PipeFactory.Create()?